components/pages/file-entity/DataAndAnalysis/index.tsx

```diff
@@ -114,15 +129,63 @@ const getWorkflowVersionLink = (workflowType: DataAnalysisWorkflowType) => {
           {workflowType.workflow_version}
         </Link>
       );
+    case WORKFLOW_NAMES.rnaSeq:
+      return (
+        <Link
+          href={`${WORKFLOW_VERSION_URLS.rnaSeq}${workflowType.workflow_version}`}
```

Comment:
we use urlJoin when we have dynamic URLs with slashes, to avoid problems with missing or too many slashes

```suggestion
          href={urlJoin(WORKFLOW_VERSION_URLS.rnaSeq, workflowType.workflow_version)}
```